regression/barebones/barebones_test.go

```diff
@@ -52,5 +53,13 @@ var _ = Describe("Barebones Test", func() {
 		err = testclient.Testclient(action, inputSpecPath)
 		Expect(err).NotTo(HaveOccurred())
 
+		By("7)Sending Caliper Invokes and Queries")
+		BENCHCFG := "benchmarks/config_barebones.yaml"
+		NETWORK := "caliper-connection-profile/caliper_connection_profile_org1.yaml"
+		benCfg := "--caliper-benchconfig"
+		netCfg := "--caliper-networkconfig"
+		caliperArgs := []string{"launch", "master", "--caliper-workspace", "./", benCfg, BENCHCFG, netCfg, NETWORK, "--caliper-flow-only-test"}
+		networkclient.ExecuteCommand("caliper", caliperArgs, true)
+
```

Comment:
In general, how long does it take to do the caliper tests, I think I'd rather duplicate the directory and move it to its own `regression/barebones_caliper` directory for now, and then add that as another stage to the daily tests

Reply:
create a separate dir, barebones_caliper, to run caliper test